### PR TITLE
Update release note to 0.11.2

### PR DESCRIPTION
**Pull request by Alima777**
### Content3 ...

<hr>

This PR has:
- [x] been self-reviewed.
    - [x] concurrent read
    - [x] concurrent write
    - [x] concurrent read and write 
- [x] added documentation for new or modified features or behaviors.
- [x] added Javadocs for most classes and all non-trivial methods. 
- [x] added or updated version, __license__, or notice information
- [x] added comments explaining the "why" and the intent of the code wherever would not be obvious 
  for an unfamiliar reader.
- [x] added unit tests or modified existing tests to cover new code paths, ensuring the threshold 
  for code coverage.
- [x] added integration tests.
- [x] been tested in a test IoTDB cluster.

<hr>

### CODE REVIEW

File: RELEASE_NOTES.md [hunk 1]
```diff
@@ -18,6 +18,26 @@
...
+# Apache IoTDB 0.11.2
+
+## Bug Fixes
+* IOTDB-1049 Fix Nullpointer exception and a delete bug in Last query
+* IOTDB-1060 Support full deletion for delete statement without where clause
+* IOTDB-1068 Fix Time series metadata cache bug
+* IOTDB-1069 restrict the flushing memtable number to avoid OOM when mem_control is disabled
+* IOTDB-1077 add insertOneDeviceRecords API in java session
+* Github issue-2169 GetObject should not return String for all data types 
+* Github issue-2240 Fix sync failed bug
```
Comment (qiaojialin) [inline RELEASE_NOTES.md:30]:
```suggestion
* Github issue-2240 fix Sync failed: Socket is closed by peer
```

File: RELEASE_NOTES.md [hunk 2]
```diff
@@ -18,6 +18,26 @@
...
 -->
+# Apache IoTDB 0.11.2
+
+## Bug Fixes
+* IOTDB-1049 Fix Nullpointer exception and a delete bug in Last query
+* IOTDB-1060 Support full deletion for delete statement without where clause
+* IOTDB-1068 Fix Time series metadata cache bug
+* IOTDB-1069 restrict the flushing memtable number to avoid OOM when mem_control is disabled
+* IOTDB-1077 add insertOneDeviceRecords API in java session
+* Github issue-2169 GetObject should not return String for all data types
```
Comment (qiaojialin) [inline RELEASE_NOTES.md:29]:
```suggestion
* Github issue-2169 GetObject returns String for all data types 
```

File: RELEASE_NOTES.md [hunk 3]
```diff
@@ -18,6 +18,26 @@
...
+
+## Bug Fixes
+* IOTDB-1049 Fix Nullpointer exception and a delete bug in Last query
+* IOTDB-1060 Support full deletion for delete statement without where clause
+* IOTDB-1068 Fix Time series metadata cache bug
+* IOTDB-1069 restrict the flushing memtable number to avoid OOM when mem_control is disabled
+* IOTDB-1077 add insertOneDeviceRecords API in java session
+* Github issue-2169 GetObject should not return String for all data types 
+* Github issue-2240 Fix sync failed bug
+* Github issue-2253 Remove unnecessary dependency
```
Comment (qiaojialin) [inline RELEASE_NOTES.md:31]: remove this

File: RELEASE_NOTES.md [hunk 4]
```diff
@@ -18,6 +18,26 @@
...
+* Add max frame size configuration
```
Comment (qiaojialin) [inline RELEASE_NOTES.md:32]:
```suggestion
* add thrift_max_frame_size in iotdb-engine.properties
```
This is not a bug

File: RELEASE_NOTES.md [hunk 5]
```diff
@@ -18,6 +18,26 @@
...
+* Fix hive-connector CI
```
Comment (qiaojialin) [inline RELEASE_NOTES.md:33]: remove this

File: RELEASE_NOTES.md [hunk 6]
```diff
@@ -18,6 +18,26 @@
...
+* Change parallelism in timeseries metadata cache
```
Comment (qiaojialin) [inline RELEASE_NOTES.md:34]: Fix the behavior of TimeseriesMetadataCache

File: RELEASE_NOTES.md [hunk 7]
```diff
@@ -18,6 +18,26 @@
...
+* Fix incorrect last result after deleting all data
+* Save one rpc call
```
Comment (qiaojialin) [inline RELEASE_NOTES.md:36]: remove this

File: RELEASE_NOTES.md [hunk 8]
```diff
@@ -18,6 +18,26 @@
...
+* Compaction not block flush
+* Cherry pick compaction modification
```
Comment (qiaojialin) [inline RELEASE_NOTES.md:38]: @zhanglingzhe0820  this should be more precise

Comment (zhanglingzhe0820) [reply RELEASE_NOTES.md:35]: There is also a pr:https://github.com/apache/iotdb/pull/2327 that Fix compaction recover block restart

Comment (zhanglingzhe0820) [reply RELEASE_NOTES.md:35]: "Cherry pick compaction modification" rename to "Fix compaction ignore modification file"

Comment (zhanglingzhe0820) [reply RELEASE_NOTES.md:35]: "Compaction not block flush" rename to "fix compaction block flush"

**Review by qiaojialin — APPROVED** [review]

**Pull request merged**